libs/stocks/feature-shell/src/lib/stocks/stocks.component.ts

```diff
@@ -32,7 +32,9 @@ export class StocksComponent implements OnInit {
     });
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    this.stockPickerForm.valueChanges.subscribe(this.fetchQuote);
+  }
 
   fetchQuote() {
     if (this.stockPickerForm.valid) {
```

Comment:
this variable won't have access to stockPickerForm as its scope will be within subscribe functions scope as its fetchQuote function is being called from within the subscribe block